src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -33,14 +33,16 @@
 
 public class Scheduler implements org.apache.mesos.Scheduler, Runnable {
     
-  public static final String ACTIVATE_MESSAGE = "activate";
+  private static final String NAMENODE_INIT_MESSAGE = "i";
+  private static final String NAMENODE_BOOTSTRAP_MESSAGE = "b";
   private static final String NAME_NODE_ID = "namenode";
   private static final String JOURNAL_NODE_ID = "journalnode";
   private static final String DATA_NODE_ID = "datanode";
   private static final String ZKFC_NODE_ID = "zkfc";
   private static final String NODE_EXECUTOR_ID = "NodeExecutor";
-  private static final String PRIMARY_NAME_NODE_EXECUTOR_ID = "PrimaryNameNodeExecutor";
-  private static final String SECONDARY_NAME_NODE_EXECUTOR_ID = "SecondaryNameNodeExecutor";
+  private static final String NAME_NODE_EXECUTOR_ID = "NameNodeExecutor";
+  private static final String NAME_NODE_TASK_STR = NAME_NODE_ID + "." + NAME_NODE_ID + "."
```

Comment:
For node id, if it's a node in the name node machine, it starts with NAME_NODE_ID followed by JOURNAL_NODE_ID, ZKFC_NODE_ID, or NAME_NODE_ID. As a side note, constants will be put in a separate file in the next PR.